index.html

```diff
@@ -471,14 +471,16 @@ <h2 class="contact">Contact me</h2>
             </label>
           </li>
           <li class="submit-container">
-            <label for="submit-btn"></label>
-            <button type="submit" placeholder="Get in touch" class="submit-btn" id="submit-btn">Get in touch</button>
+            
+            <p class="message">Wrong email</p>
+            <button type="" placeholder="Get in touch" class="submit-btn" id="submit-btn">Get in touch</button>
```

Comment:
- [ ] Instead of displaying the `Wrong email` error message, Please display some meaningful error message so users understand what is wrong and how they can fix it. It will improve the user experience.
  - You can consider this error message `Please enter email address in lower case.`